runtime-modules/content/src/lib.rs

```diff
@@ -674,10 +674,13 @@ decl_module! {
 
             // upload to storage
             if let Some(upload_assets) = params.assets.as_ref() {
+
+        // upload assets to storage after creating eth bag
                 Self::upload_assets_to_storage(
                     upload_assets,
                     &channel_id,
                     &sender,
+                    true,
                 )?;
             }
 
```

Comment:
Is it correct?
```
// a newly create channel has zero videos ??
num_videos: 0u64,
```

Reply:
What's about this comment?

Reply:
Nothing really, the only way to increase video counter for a channel is by the `create_video` extrinsic

Reply:
Mokhtar confirmed that this comment should be deleted.